tests/unit/CMakeLists.txt

```diff
@@ -368,3 +368,10 @@ add_custom_target(test_lcp ALL DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/test_lcp)
 add_test(test_lcp ${CMAKE_CURRENT_BINARY_DIR}/test_lcp)
 add_dependencies(memgraph__unit test_lcp)
 
+# Test websocket
+if (MG_ENTERPRISE)
```

Comment:
Why on `enterprise`?

Reply:
Since on community the tests will not work for the authorization part, that part only works in enterprise

Reply:
Will remove since a better solution was suggested, by just skipping the authorization test